lib/services/ngsi/entities-NGSI-LD.js

```diff
@@ -540,7 +540,7 @@ function sendUpdateValueNgsiLD(entityName, attributes, typeInformation, token, c
         }
     }
 
-    payload = NGSIUtils.castJsonNativeAttributes(payload);
+    payload = NGSIUtils.castJsonNativeAttributesTypes(payload);
```

Comment:
After @jason-fox [comment](https://github.com/telefonicaid/iotagent-node-lib/pull/1324/files#r1145876564), this change can be reverted having both version (V2 and LD ) same behaviour 

```suggestion
    payload = NGSIUtils.castJsonNativeAttributes(payload);
```

Reply:
+ the duplicated function removed in NGSIUtils, if no longer used.